source/dagger/gameplay/team_game/hero_character.cpp

```diff
@@ -0,0 +1,6 @@
+#include "hero_character.h"
+
+hero_character::hero_character()
```

Comment:
This...
a) isn't needed because the class doesn't exist
b) doesn't compile because of the class not existing